personality/voms/files/log4j-trustmanager.properties

```diff
@@ -0,0 +1,7 @@
+log4j.logger.org.glite.security=INFO, fileout
```

Comment:
Pretty sure ncm-metaconfig can write `.properties` files, but this one doesn't seem to be used in this PR.